zowe/imperative
# feat: Use `keyring` module in `@zowe/secrets-for-zowe-sdk` as replacement for `node-keytar`

**What It Does**

This PR implements the use of the `keyring` module from the new `@zowe/secrets-for-zowe-sdk`. It also replaces the use of `node-keytar` as a result.

**Note:** Once approved, I will publish a release of Secrets SDK and then update this PR to point to the production version.

**How to Test**

Reviewers should be able to link this PR with the latest changes on the CLI `master` branch (using `npm run link:imperative`), and then install/use the CLI to store credentials (e.g., `zowe config secure --gc`) as if nothing has changed.

**Review Checklist**
I certify that I have:
